test/torch/nn/test_conv.py

```diff
@@ -4,6 +4,10 @@
 import pytest
 
 
+# Disable mkldnn to avoid rounding errors due to difference in implementation
+th._C._set_mkldnn_enabled(False)
```

Comment:
MAYBE: Wouldn't be better to unset the ```mkldnn``` and then set it to ```True``` again in the function.

Reply:
I agree, I will change it